### PR TITLE
limiting homepage episodes to 'featured'

### PR DESCRIPTION
**Pull request by tomhazledine**

### CODE REVIEW

**Comment by edhaz** [comment]
Looks good, however:
- I feel like maybe there should be five featured episodes?
- How did you select the featured episodes?
- Should the episode numbers now be shown on the Archive page? Bit hard to parse as it currently is (for me anyway).

**Comment by tomhazledine** [comment]
A the moment, any post with `featured: true` in the frontmatter will appear on the homepage. I'll set a max-limit of `5`, but you can add that frontmatter to any post you want to include.

I think you're right - the full archive needs a bit of love. Just a dull hard-to-parse list at the moment 😬

**Comment by tomhazledine** [comment]
I'll revisit the archive page sometime soon, but for now I've set it to use the EpisodeSummary component (like the list on the homepage) so the list is a little easier to read.

Also limited the "featured" list to show a max of 5 featured episodes. Feel free to add/remove the "featured" frontmatter from whichever episodes you like. I just picked a few that had above-average "first 30 days" download numbers.

I think this PR is ready to merge now 😎
